indico/modules/events/layout/default.py

```diff
@@ -178,7 +178,7 @@ def get_default_menu_entries():
         MenuEntryData(
             title=_("My Sessions"),
```

Comment:
Also pass the `visible` callback and only show the menu items if you have any sessions that will be shown in "my sessions"